lib/qldb.connect.js

```diff
@@ -311,7 +311,7 @@ class qldbConnect {
         }
         // Get the WHERE part for the query
         const sqlWhere = await this.createSqlWhere(args, tableName, model);
-        return 'SELECT ' + sqlSelectFields + ' FROM ' + tableName + onStatement  + sqlWhere +';';
+        return 'SELECT ' + sqlSelectFields + ' FROM ' + tableName + ' BY documentId ' + onStatement + sqlWhere + ';';
```

Comment:
Why are you adding the documentid?

Reply:
Wanted to return documentId also in get query data.
We can only retrieve documentId in select fields if we add by clause in query

Reply:
Yes. so, shall we do that way then by checking for that argument and build query according to that?

Reply:
Sounds like a good solution.